public/locales/cs/explorer.json

```
@@ -1,4 +1,22 @@
 {
+  "meta": {
```

Biome (2.1.2)

[error] The key meta was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)